### PR TITLE
[WIP] Removing snowplow 

### PR DESCRIPTION
**Pull request by dominickendrick**
## Why are you doing this?
This removes the calls to the `event-stream-analytics` tracking pipeline AKA `snowplow`.
This system is being deprecated in favour of using the acquisitions pipeline in Ophan.

image

This should only be merged once we have confirmed all new tracking works correctly

@guardian/contributions

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -17,7 +17,6 @@ object Dependencies {
   val playCache = PlayImport.cache
   val awsSimpleEmail = "com.amazonaws" % "aws-java-sdk-ses" % awsClientVersion
   val sqs = "com.amazonaws" % "aws-java-sdk-sqs" % awsClientVersion
-  val snowPlow = "com.snowplowanalytics" % "snowplow-java-tracker" % "0.5.2-SNAPSHOT"
```
Comment (rtyley) [inline project/Dependencies.scala]: Now that this dependency is being removed you can also remove the 'snapshots' resolver! :confetti_ball: 

It was originally added here... https://github.com/guardian/membership-frontend/pull/191/files#r152640638

File: project/Dependencies.scala [hunk 2]
```diff
@@ -17,7 +17,6 @@ object Dependencies {
...
   val bCrypt = "com.github.t3hnar" %% "scala-bcrypt" % "3.1"
```
Comment (rtyley) [inline project/Dependencies.scala]: Do we still need `bCrypt`?

Comment (dominickendrick) [reply project/Dependencies.scala]: No, I'll remove it : )

File: project/Dependencies.scala (hunk 1 above)
Comment (dominickendrick) [reply project/Dependencies.scala]: Done!

**Comment by Mullefa** [comment]
As discussed, Bolt uses a data source dependant on Snowplow to report on supporters. Therefore, we should check it is ok to stop reporting on supporters in Bolt before merging this.

**Comment by johnduffell** [comment]
is this going anywhere? there's a nice set of conflicts here!

**Comment by davidfurey** [comment]
@johnduffell - I hope so, this this will allow us to block non-https traffic for membership-frontend, ticking another thing off CyberEssentials.

**Comment by johnduffell** [comment]
🎉

**Pull request merged**